junit/MatrixTest.java

```diff
@@ -362,6 +361,83 @@ public void testGetException()
         testIndex[1] = -1; 
         matrix.get(testIndex[0], testIndex[1]);
     }
+    /**
+     * testScalarMult -Test for the scalar times function.
+     */
+    @Test
+    public void testScalarMult() 
+    {
+        Random rand = new Random();
+        int multiplier = rand.nextInt(100);
+        double[][] testMatrix = {
+            {1, 2, 3, 4},
+            {5, 6, 7, 8},
+            {9, 10, 11, 12}
+        };
+        double[][] corrMatrix = new 
+            double[testMatrix.length][testMatrix[0].length];
+        for (int i = 0; i < testMatrix.length; i++) 
```

Comment:
I think that we might want to hard-code the correct matrix result here instead of computing it. By computing it, we replicate the logic from within the method that we are testing. It doesn't really give us that "double-check" validation that Dr. Fenwick was talking about, because we aren't arriving at the same answer two different ways. 
@ccampell @bclinthall Thoughts?